galaxy.yml

```diff
@@ -37,12 +37,12 @@ tags:
 
 dependencies:
   'git+https://github.com/cloudera-labs/cloudera.cloud.git': 'main'
-  'ansible.netcommon': '>=2.0.2'
-  'community.aws': '>=1.2.0'
-  'community.general': '>=3.1.0'
-  'amazon.aws': '>=1.3.0'
-  'azure.azcollection': '>=1.4.0'
-  'google.cloud': '>=1.0.2'
+  'ansible.netcommon': '2.5.1'
+  'community.aws': '3.0.1'
+  'community.general': '4.5.0'
+  'amazon.aws': '3.0.0'
+  'azure.azcollection': '1.11.0'
+  'google.cloud': '1.0.2'
   # netapp.azure
   # Bug fix for Netapp collection
   'git+https://github.com/ansible-collections/netapp.git#ansible_collections/netapp/azure': 'main'
```

Comment:
I believe thisbug fix is no longer needed and netapp.azure should be pinned as a part of this change.